tensorbay/client/version.py

```diff
@@ -599,10 +599,16 @@ def _create_job(
             arguments: The arguments dict of the specific job.
             description: The Job description.
 
-        Return:
+        Returns:
             The id of the job.
 
         """
+        post_data: Dict[str, Any] = {"title": title, "jobType": job_type, "arguments": arguments}
+        if description:
+            post_data["description"] = description
+        response = self._client.open_api_do("POST", "jobs", self._dataset_id, json=post_data).json()
+        job_id: str = response["jobId"]
+        return job_id
```

Comment:
should `_create_job` return a Job instance? or a dict contains job details for subclass to return its Job instance?

Reply:
We could get job instance by using get_job in create_squash_and_merge_job. Open-api just need to return id like `create_draft`, `create_dataset`.

Reply:
Does using `get_job` mean we need to send one more request?

Reply:
Yes, but I think all the job information should be obtained through the get_job interface. This will facilitate our management. We do not need to write SquashAndMergeJob.load() in squash_and_merge.get_job and squash_and_merge.create_job, even the squash_and_merge.abort_job(if we need to get job instance).

Reply:
1. I cannot get the relation between `load` and `create_job`
2. I also think the `abort` and `retry` should be put into job instance:
```python
job = xxx.create_job()
job.update()
job.retry()
job.abort()
```

Reply:
Ok, the job.retry and job.abort will be redesigned in the https://github.com/Graviti-AI/tensorbay-python-sdk/pull/1134